newm-server/src/main/kotlin/io/newm/server/features/minting/repo/MintingRepositoryImpl.kt

```diff
@@ -400,6 +393,7 @@ class MintingRepositoryImpl(
 
     @VisibleForTesting
     internal fun buildStreamTokenMetadata(
+        release: Release,
```

Comment:
I'm curious why would you pass `release`  to this method `buildStreamTokenMetadata` instead of fetching it within method scope since you are already pass `song` object

Reply:
I believe that I needed it in the method that calls this one so it was already retrieved.